googleapiclient/discovery.py

```diff
@@ -315,6 +315,15 @@ def build_from_document(
 
   if isinstance(service, six.string_types):
     service = json.loads(service)
+
+  if  'rootUrl' not in service and (isinstance(http, HttpMock) or
```

Comment:
No need for two `isinstance` calls; [you can just pass both classes to a single call](https://docs.python.org/2/library/functions.html#isinstance):
```
isinstance(http, (HttpMock, HttpMockSequence,))
```
.